modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -2605,6 +2605,16 @@ export const console: ConsoleNS = {
                             message: "Cannot add to this step"
                         }
                     },
+                    tierLimitReachedError: {
+                        emptyPlaceholder: {
+                            action: "View Plans",
+                            subtitles: "You can contact the organization administrator or (if you are the " +
```

Comment:
Nothings wrong with the spellings. But I didn't get `if you are the administrator` part

Reply:
Upgrading subscriptions can only be done by organisation admin. This modal can be displayed for both admins and non-admin users. Therefore the message is displayed here in a generic manner.